src/Workspaces/CSharp/Portable/CodeGeneration/CSharpSyntaxGenerator.cs

```diff
@@ -540,12 +521,12 @@ private SyntaxNode WithBodies(SyntaxNode declaration)
             return declaration;
         }
 
-        private AccessorListSyntax WithBodies(AccessorListSyntax accessorList)
+        private static AccessorListSyntax WithBodies(AccessorListSyntax accessorList)
         {
-            return accessorList.WithAccessors(SyntaxFactory.List(accessorList.Accessors.Select(a => WithBody(a))));
+            return accessorList.WithAccessors(SyntaxFactory.List(accessorList.Accessors.Select(WithBody)));
```

Comment:
:bulb: It's even more efficient to use a cached delegate which avoids calling through the intermediate lambda.